HyperwalletUISDK.xcodeproj/project.pbxproj

```diff
@@ -4413,7 +4429,10 @@
 				DYLIB_COMPATIBILITY_VERSION = 1;
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
-				FRAMEWORK_SEARCH_PATHS = "$(PROJECT_DIR)/Carthage/Build/iOS";
+				FRAMEWORK_SEARCH_PATHS = (
+					"$(PROJECT_DIR)/Carthage/Build/iOS",
+					"$(PROJECT_DIR)",
```

Comment:
why this change?

Reply:
Due to change in latest refactored code, these fixes has been done in following PR: https://github.com/hyperwallet/hyperwallet-ios-ui-sdk/pull/167